app/org/iatoki/judgels/sealtiel/services/ClientAcquaintancesService.java

```diff
@@ -0,0 +1,8 @@
+package org.iatoki.judgels.sealtiel.services;
+
+public interface ClientAcquaintancesService {
```

Comment:
The second parameter for both problem should be `acquaintanceJid` for consistency.